src/components/layout/ContentWrapper.tsx

```diff
@@ -0,0 +1,24 @@
+import { DEFAULT_LAYOUT_MAXWIDTH } from '@/configs/constants'
+import { Stack } from '@mui/material'
+import type { PropsWithChildren } from 'react'
+
+/**
+ * Wrapper for page content to set a consistent layout and max width
+ */
+export const ContentWrapper = ({ children }: PropsWithChildren) => {
```

Comment:
Jeg kan ta feil her, men lurer på om dette er et anti pattern for next app router selv om det er god klassisk react måte å gjøre det på.

årsaken er at man kan ha layout.tsx som en fil i app hierarkiet som gjør dette her (og dermed prosesseres annerledes, f.eks ved at layout blir statisk generert på build og kun innholdet er server generert for raskere lasting). Har ikke helt kommet meg inn i app router ennå så kan være dette er riktig metode, men om du gjenbruker det på nesten samtlige pages så hører det enten hjemme på layout i root nivå i app, eller et annet sted.

https://nextjs.org/docs/pages/building-your-application/routing/pages-and-layouts#layout-pattern